pkg/fileservice/local_fs.go

```diff
@@ -147,8 +147,19 @@ func (l *LocalFS) Write(ctx context.Context, vector IOVector) error {
 	default:
 	}
 
-	ctx, span := trace.Start(ctx, "LocalFS.Write")
-	defer span.End()
+	var err error
+	sCtx, span := trace.Start(ctx, "LocalFS.Write", trace.WithKind(trace.SpanKindLocalFSVis))
```

Comment:
should overwrite ctx here instead of introducing sCtx.

Reply:
fixed